frame/bounties/src/tests.rs

```diff
@@ -54,12 +54,12 @@ frame_support::construct_runtime!(
 );
 
 parameter_types! {
-	pub const BlockHashCount: u64 = 250;
 	pub const MaximumBlockWeight: Weight = 1024;
```

Comment:
this can be removed no ? actually it seems not used

Reply:
Alright

Reply:
```suggestion
```